hawkpost/templates/account/auth_modals.html

```diff
@@ -98,15 +88,27 @@ <h3 class="title"> Sign up</h3>
             </label>
           </div>
           {% if redirect_field_value %}
-            <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
+          <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}" />
           {% endif %}
         </div>
         <div class="col-xs-12">
-          <button class="btn-default" type="submit">{% trans "Sign Up" %}</button>
+          <button class="btn-default mt-small" type="submit">{% trans "Sign Up" %}</button>
+        </div>
+        <div class="col-xs-12 middle-xs flex-in">
+          <p class="no-padding">Sign up using:</p>
+          <ul class="no-padding">
+            {% include "socialaccount/snippets/provider_list.html" with process="login" %}
```

Comment:
I think this include is dependent on the conditional of line 72 (`{% if socialaccount_providers %}`), no? probably you should also check it here.